### PR TITLE
merge template updates v2.5.1

### PR DESCRIPTION
**Pull request by ggabernet**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/airrflow/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/airrflow _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,21 +3,26 @@
...
-## [2.2.1] - dev
+## [2.3.0] - dev
 
 ### `Added`
 
 - [#180](https://github.com/nf-core/airrflow/pull/180) Added possibility to add any property in the AIRR sequence table as label on the lineage tree nodes.
 - [#180](https://github.com/nf-core/airrflow/pull/180) Lineage tree construction now also includes trees with just one sequence.
 - [#180](https://github.com/nf-core/airrflow/pull/180) Added metadata annotation to final repertoire table.
 - [#180](https://github.com/nf-core/airrflow/pull/180) Added possibility to provide custom Rmarkdown report.
+- [#183](https://github.com/nf-core/airrflow/pull/183) Update template to nf-core tools v2.5.2
```
Comment (maxulysse) [inline CHANGELOG.md:14]: `2.5.2`?
Isn't it `2.5.1`

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,21 +3,26 @@
...
 ### `Fixed`
 
 - [#180](https://github.com/nf-core/airrflow/pull/180) Repertoire analysis report now also saves diversity table.
 
 ### `Dependencies`
 
+| Dependency | Old version | New version |
+| ---------- | ----------- | ----------- |
+| multiqc    | 1.12        | 1.13        |
+
```
Comment (maxulysse) [inline CHANGELOG.md:25]: @FriederikeHanssen We should do that

**Review by maxulysse — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (ggabernet) [reply CHANGELOG.md:14]: yes, very well seen, thanks!

**Pull request merged**